client/my-sites/plans/controller.jsx

```diff
@@ -87,6 +87,10 @@ export default {
 			basePath = route.sectionify( context.path );
 		let baseAnalyticsPath;
 
+		if ( config.isEnabled( 'manage/plan-features' ) ) {
```

Comment:
Whoa, prod is doing a few crazy redirects atm. 

Since this redesign is a merging of both /plans and /plans/compare it makes sense (if the feature flag is enabled) to render this new component for the following routes, and not redirect:
`/plans/features/:feature/:domain` (no redirect)
`/plans/compare/:domain` (no redirect)
`/plans/:domain` (no redirect)

As @rralian discussed before, we can avoid highlighting on the `/plans/features/:feature/:domain` route.